dashboard/db/migrate/20151022174100_create_puzzle_ratings.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePuzzleRatings < ActiveRecord::Migration
+  def change
+    create_table :puzzle_ratings do |t|
+      t.references :user, index: true
```

Comment:
this index is not necessary because you also have user_id as the first column in a multi-column index